onebusaway-android/src/google/java/org/onebusaway/android/map/googlemapsv2/BaseMapFragment.java

```diff
@@ -346,6 +349,23 @@ protected void showDialog(int id) {
         MapDialogFragment.newInstance(id, this).show(getFragmentManager(), MapDialogFragment.TAG);
     }
 
+    private void setLocationByZip(final String zip) {
+        Runnable r = new Runnable() {
```

Comment:
This should be posted to an Android `Handler` instead of just running a Thread - for example, something like:

~~~
final Handler h = new Handler();
h.post(new Runnable() {...
~~~